mindbender/maya/plugins/validate_project_editorial_info.py

```diff
@@ -0,0 +1,50 @@
+# validate_project_editorial_info.py
+import pyblish.api
+
+
+class ValidateMindbenderProjectEditInfo(pyblish.api.ContextPlugin):
+    """
```

Comment:
Have a look at this docstring, and make sure it's in [Napoleon](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) format.

Code convention [here](https://github.com/mindbender-studio/core#code-convention)